test/runtests.jl

```diff
@@ -320,6 +321,15 @@ Unitful.uconvert(U::Unitful.Units, q::QQQ) = uconvert(U, Quantity(q.val, cm))
 
 @testset "Promotion" begin
     @testset "> Unit preferences" begin
+        # Should warn on possible redundant units issue (ms and s)
+        @test_logs (:warn,) Unitful.preferunits(C/ms)
```

Comment:
I'd check the message with something like
```suggestion
        @test_logs (:warn, r"^Preferred units contain complex units") Unitful.preferunits(C/ms)
```
to make sure we aren't catching something else by accident (even though it's very unlikely)